hc-tree-view-behavior.html

```diff
@@ -86,6 +87,9 @@
     itemStateChanged(event, detail) {
       const {item, state} = detail;
 
+      // Reject duplicated events
+      if (item.state === state) return;
```

Comment:
This is a culling maneuver to remove calls all together when they are irrelevant, in a debounce it still eventually performs the task.  I plan on adding some debouncing later though to help with some of the duplicated calls that culling won't prevent though.